assets/src/util/assignment.js

```diff
@@ -26,7 +26,7 @@ const setAssignmentGoalGrade = (key, assignments, goalGrade) => {
     {
       ...assignments[key],
       goalGrade: goalGrade === '' ? '' : roundToXDecimals(Number(goalGrade), 3),
-      goalGradeSetByUser: goalGrade
+      goalGradeSetByUser: !!goalGrade
```

Comment:
Sam I think this is the change that should fix this issue? Sorry I think I may have incorrectly merged the master in the previous commit - this one should work.

Reply:
Cool, thanks, I'll try this in the next couple hours and review.